subprojects/core/src/main/java/org/gradle/initialization/ParallelConfiguration.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.initialization;
+
+import org.gradle.api.Incubating;
+
+/**
+ * A {@code ParallelConfiguration} defines the parallel settings for a Gradle build.
+ */
+public interface ParallelConfiguration {
```

Comment:
I think `ParallelismConfiguration` is more meaningful than `ParallelConfiguration`. The latter can be misinterpreted to "configuration in parallel" which isn't what we're up to here.